src/Sentry.Unity.Editor/Android/AndroidManifestConfiguration.cs

```diff
@@ -133,17 +147,22 @@ internal void SetupSymbolsUpload(string basePath)
             var options = _getOptions();
             var logger = options?.DiagnosticLogger ?? new UnityLogger(new SentryUnityOptions());
 
-            var sentryCliOptions = SentryCliOptions.LoadCliOptions();
-            if (!sentryCliOptions.UploadSymbols)
+            if (_scriptingImplementation != ScriptingImplementation.IL2CPP)
+            {
+                logger.LogDebug("Automated symbols upload requires the IL2CPP scripting backend.");
+                return;
+            }
+
+            var sentryCliOptions = _getSentryCliOptions();
+            if (sentryCliOptions is null || !sentryCliOptions.UploadSymbols)
```

Comment:
In the event `null` is returned, it'll be hard to tell from the log message. Ideally we'd have to `if`s here, one could be a `Warn` or `Err`? Why/When would we return `null`?

Reply:
There should not be a case where it's `null`. If we can't find it at the default location we create a new one and return that one. Which might be its own issue.
I'll separate the null-check to have it covered anyway as I intend to refactor parts of our scriptable object creation and the editor config window when we improve our onboarding experience.